addons/google_gmail/i18n/google_gmail.pot

```diff
@@ -216,14 +216,6 @@ msgstr ""
 msgid "The URL to generate the authorization code from Google"
 msgstr ""
 
-#. module: google_gmail
```

Comment:
Should we really remove the translation? We usually keep it in case code is not updated.

It will be automatically removed upon next export (saas version) anyway.